website1/css/base.css

```diff
@@ -0,0 +1,15 @@
+body {
+  font-family: "Source Sans Pro", sans-serif;
+}
+
+.container {
+  max-width: 1440px;
+}
+
+* {
+  font-family: "Source Sans Pro", sans-serif;
+  font-weight: 400;
+  font-size: 18px;
+  text-decoration: none;
```

Comment:
This property is not required for all elements.
Set this property only for tags that you need.